src/main/java/Angela.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+import java.util.List;
```

Comment:
```suggestion
import java.util.List;
import java.util.Scanner;
```
Perhaps your imports could be ordered lexicographically?